ingestion/functions/parsing/common/python/parsing_lib/parsing_lib.py

```diff
@@ -43,21 +43,13 @@ def write_to_server(cases, headers):
     Upserts the provided cases via the G.h Case API.
 
     TODO: Link out to the Case API resource documentation, once available.
```

Comment:
This TODO is also closable (though the link won't be for prod).

Reply:
removed.